web/tests/user_pill.test.cjs

```diff
@@ -73,29 +66,29 @@ function test(label, f) {
 }
 
 test("create_item", ({override}) => {
-    function test_create_item(email, current_items, expected_item, pill_config) {
-        const item = user_pill.create_item_from_email(email, current_items, pill_config);
+    function test_create_item(user_id, current_items, expected_item, pill_config) {
+        const item = user_pill.create_item_from_user_id(user_id, current_items, pill_config);
         assert.deepEqual(item, expected_item);
     }
 
     settings_data.user_can_access_all_other_users = () => false;
 
-    test_create_item("bogus@example.com", [bogus_item], undefined);
-    test_create_item("bogus@example.com", [], undefined);
```

Comment:
could you still do a bogus item with an invalid id?

Reply:
Those will throw an error or we will create a fake object for them with `is_inaccessible_user` flag. `is_valid_user_id` is the new source of truth.

Reply:
Ah ok, thanks for explaining!